index.html

```diff
@@ -14,10 +14,11 @@
             <div class="logo">
                 <a class="a_logo" href="#">My Logo</a>
             </div>
-            <nav class="menu">
-                <a href="#">
-                    <img src="./images/Icon-Menu.png" alt="Menu" class="menu-icon"/>
-                </a>
+            <nav>
+                <a class="nav-links" href="#">Portfolio</a>
+                <a class="nav-links" href="#">About</a>
+                <a class="nav-links" href="#">Contact</a>
+                <a id="metaMenu" href="#">&#9776;</a>
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you wrap `all your nav list` in a `ul` tag and each `<a>` link into in `li` tag for good HTML semantics. e.g. :
 ```bash
                <ul>
                    <li><a href="">Portfolio</a></li>
                    <li><a href="">Portfolio</a></li>
                   .....
                </ul>
```

- [ **OPTIONAL** ] Also make sure to `center` the content of the headline section and kindly check the `spacing` at the **top**, **bottom**, **right**, and **left** of the section(headline-section), `font-size`, `line height`,etc. Please, I suggest you fix this, as the previous reviewer forgot to point it out.
## Figma design
![home_f](https://user-images.githubusercontent.com/83042742/171699848-16ddc692-75b3-49b3-84ba-588ce4d707ce.png)

## Your design
![Home_s](https://user-images.githubusercontent.com/83042742/171699869-8ce4ecdf-65b8-45da-972a-7be4c6c89b68.png)

#### Don't care about borders around texts

Reply:
@tresorsawasawa Thanks for the detailed feedback. You realise that once I start adding width and height, the page doesn't exactly fit the mobile. Kindly verify. Thank you